apps/meteor/client/NavBarV2/NavBarSearch/NavBarSearchUserRow.tsx

```diff
@@ -0,0 +1,37 @@
+import { SidebarV2ItemIcon } from '@rocket.chat/fuselage';
+import type { SubscriptionWithRoom } from '@rocket.chat/ui-contexts';
+import { useSetting } from '@rocket.chat/ui-contexts';
+import type { ComponentProps, ReactElement } from 'react';
+import { memo } from 'react';
+
+import NavBarSearchItem from './NavBarSearchItem';
+import { ReactiveUserStatus } from '../../components/UserStatus';
+import { roomCoordinator } from '../../lib/rooms/roomCoordinator';
+
+type NavBarSearchUserRowProps = {
+	room: SubscriptionWithRoom;
+	id: string;
+	style?: CSSStyleRule;
+	AvatarTemplate: ReactElement;
+} & Partial<ComponentProps<typeof NavBarSearchItem>>;
+
+const NavBarSearchUserRow = ({ room, id, style, AvatarTemplate, ...props }: NavBarSearchUserRowProps) => {
```

Comment:
I don't see this `style` prop being passed

Reply:
fixed!